src/Benchmarks/Data/EfDb.cs

```diff
@@ -70,9 +71,13 @@ public async Task<World[]> LoadMultipleUpdatesRows(int count)
             return results;
         }
 
+#if NETCOREAPP2_1 || NETCOREAPP2_2
         private static readonly Func<ApplicationDbContext, AsyncEnumerable<Fortune>> _fortunesQuery
             = EF.CompileAsyncQuery((ApplicationDbContext context) => context.Fortune);
-
+#else
+        private static readonly Func<ApplicationDbContext, IAsyncEnumerable<Fortune>> _fortunesQuery
```

Comment:
Funny, I was working on the PR for TechEmpower to move to preview7, and fought some time with this change trying to figure out what to do, and did exactly the same change with the new extension method.

Reply:
Yeah, compiled query wouldn't have worked on preview7 :)